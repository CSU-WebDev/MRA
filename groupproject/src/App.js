import Content from "./components/Content";
import Footer from "./components/Footer";
import Top from "./components/Top";
import React,{useState} from "react";

function App() {
  const [data, setdata] = useState('');
  return (
    <div className="App">
      <Top />
      <Content setdata={setdata}  />
      <Footer />
    </div>
  );
}

export default App;
