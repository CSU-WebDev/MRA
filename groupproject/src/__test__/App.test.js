import React from 'react';
import App from '../App';
import {render} from '@testing-library/react';
import userEvent from '@testing-library/user-event';

test('renders learn react link', () => {
  const {asFragment} = render(<App />);
        expect(asFragment()).toMatchSnapshot()
});
