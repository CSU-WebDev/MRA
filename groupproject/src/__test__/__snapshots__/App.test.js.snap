// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      id="Nav"
    >
      <div
        class="justify-content-center nav"
      >
        <h1>
           Weather Info
        </h1>
        <div
          class="nav-item"
        >
          <a
            class="nav-link"
            data-rr-ui-event-key="#App-header"
            href="#App-header"
            id="Nav"
          >
            Home
          </a>
        </div>
        <div
          class="nav-item"
        >
          <a
            class="nav-link"
            data-rr-ui-event-key="#footer"
            href="#footer"
            id="Nav"
          >
            Team
          </a>
        </div>
        <div
          class="nav-item"
        >
          <a
            class="nav-link"
            href="#"
            id="Nav"
            role="button"
            tabindex="0"
          >
             Change Mode 
          </a>
        </div>
      </div>
    </div>
    <header
      class="App-header"
    >
      <div
        class="d-flex justify-content-center"
      >
        <label
          for="city"
        >
          Zip Code : 
        </label>
        <input
          id="city"
          placeholder="Example: 31907"
          type="text"
        />
        <button
          class="btn btn-warning"
          type="button"
        >
          Submit
        </button>
      </div>
      <br />
      <div
        class="customCardList scrolls card-group"
      />
    </header>
    <div
      id="footer"
    >
      <p
        id="footer"
      >
        MRA Â© May 2023 
      </p>
      This is a group project by :
      <p
        id="footer"
      >
        Adeola Ogundipe
      </p>
      <p
        id="footer"
      >
        Raj Kotak
      </p>
      <p
        id="footer"
      >
        Mckenna Koetitz
      </p>
    </div>
  </div>
</DocumentFragment>
`;
